packages/terra-clinical-item-display/CHANGELOG.md

```diff
@@ -6,6 +6,9 @@
   * `strong` tag around strong style text.
   * `span` tag around non-strong style text.
 
+* Changed
+  * Make screen reader support for text styling more robust by using terra-visually-hidden-text.
```

Comment:
Nitpicky but how does this sound?
```suggestion
  * Improved screen reader support for text styling by utilizing terra-visually-hidden-text.
```

Reply:
Done: https://github.com/cerner/terra-clinical/pull/855/commits/106c28265abb46338f0d525e4e02e91657637d29